engine/table/src/main/java/io/deephaven/engine/table/impl/MultiJoinModifiedSlotTracker.java

```diff
@@ -8,8 +8,8 @@
 import java.util.List;
 
 public class MultiJoinModifiedSlotTracker {
-    /** The slot that was modified. */
-    private final IntegerArraySource modifiedSlots = new IntegerArraySource();
+    /** The main table row that was modified. */
```

Comment:
Talking about main table in this class is misleading. It is only ever concerned with output row position. There are a few comments that need updating.